### DISCUSSION TITLE
Modbus TCP IP Connection not working anymore

### DISCUSSION THREAD

**Discussion by Puth12** [wlcrs/huawei_solar#789] [Setting up the integration]
Hi,
I have previously used EVCC with Modbus Connection to my Huawei Network Port 502 (s Dongle A) which was perfectly working.

Since I wanted to use Home Assistant i have removed Evcc from my Raspberry PI and installed HA.

Then I tried to use this huawei solar Integration with Network. 
Unfortunately it is now not working to use the Network Connection in that Integration to the same IP Adress (fixed in my Network, to Dongle). I use same Port 502 as in the Evcc configuration. Slave ID 1.

I tried port 6607 and Slave ID 11 or 16. combinations of it are also not working.

could you please help me?

thanks a Lot

Br

**Comment by Roving-Ronin** [comment]
Have you had a look at the Wiki and also checked to see if others have your gear working, and if so what firmware versions etc?
https://github.com/wlcrs/huawei_solar/wiki/Confirmed-firmware-versions

Also confirmed the slave ID's, 11 / 16 seem rather 'high'. Normally from factory will be 1 or 2 etc. Also try port 502.

**Reply by Puth12** [reply]
Thanks, i will Check the firmwares but I have tried Slave ID 1 and Port 502. it was not working

**Reply by Puth12** [reply]
I have checked but unfortunately I couldn’t find an entry for SUN2000-M1 10KTL with Wifi sDongleA. Because I have used Modbus TCP IP 502 (ID 1) with the Wifi sDongleA.

**Reply by Roving-Ronin** [reply]
Look at this line:

<html><body>


<html><body>


Inverter model | Firmware | MBUS | Smart Dongle | Firmware | LUNA ESS | Firmware | Connection Mode | Port | Slave | Comment
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
SUN2000-6KTL-M1 | V100R001C00SPC160 | SPC330 | sDongleA-05 | V200R022C10SPC110 | No | N/A | Option B | 502 | 2,1 | Dongle WiFi via home network

</body>
</html>

It might be a 6KTL M1 but its the same otherwise as your 10MTL M1.
